File: URI1036.py
from math import sqrt

numeros = input()
A, B, C = numeros.split(" ")
A = float(A)
B = float(B)
C = float(C)
if A == 0:
    print("Impossivel calcular")
else:
    delta = (B ** 2) - 4 * A * C
    if delta >= 0:
        R1 = (-B + sqrt(delta)) / (2 * A)
        R2 = (-B - sqrt(delta)) / (2 * A)
        print("R1 = {:.5f}".format(R1))
        print("R2 = {:.5f}".format(R2))
    else:
        print("Impossivel calcular")

File: C++/URI1036.cpp
#include <iostream>
#include <iomanip>
#include <cmath>
using namespace std;

int main () {
    double a, b, c, delta;
    cin >> a >> b >> c;
    delta = (pow(b,2) - (4 * a * c));

    if (a == 0 || delta < 0)
        cout << "Impossivel calcular" << endl;
    else {
        double r1, r2;
        r1 = (-b + sqrt(delta))/(2*a);
        r2 = (-b - sqrt(delta))/(2*a);
        cout << fixed << setprecision(5);
        cout << "R1 = " << r1 << endl;
        cout << "R2 = " << r2 << endl;
    }


}
